src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/index.tsx

```diff
@@ -401,7 +407,7 @@ class IssueRuleEditor extends AsyncView<Props, State> {
     ];
 
     const {rule, detailedError} = this.state;
-    const {actionMatch, actions, conditions, frequency, name} = rule || {};
+    const {actionMatch, actions, filters, conditions, frequency, name} = rule || {};
```

Comment:
I know this isn't your code, but I don't think we need to destructure actionMatch out of the rule object since it just gets added back in to initialData at line:428 and I don't see any other place where actionMatch is used.